flags/flags.go

```diff
@@ -79,12 +79,14 @@ func ParseFlagsFromArgsOrDie(appname string, data interface{}, args []string, ad
 	return ""
 }
 
-// ActiveCommand returns the name of the currently active command.
+// ActiveCommand returns the concatenation of all subcommands that make up the command.
```

Comment:
This makes it a breaking change - seems probably worth adding a new function with the new behaviour (and maybe marking this one deprecated) since we have a few different things using this.

Reply:
Yeah, I bumped the version to `v6` in go.mod. I think this should be fine? Unless we follow a different approach to breaking changes? As in, the same function name shouldn't be reused if we are going to break it.